go/chat/push.go

```diff
@@ -473,6 +511,26 @@ func (g *PushHandler) Activity(ctx context.Context, m gregor.OutOfBandMessage) (
 			if g.badger != nil && nm.UnreadUpdate != nil {
 				g.badger.PushChatUpdate(*nm.UnreadUpdate, nm.InboxVers)
 			}
+		case "setAppNotificationSettings":
```

Comment:
Yeah I should just define them in client and vendor them into Gregor, I'll do that now. Should have just made this an enum from the get-go, not sure why I didn't, but too annoying to change now.